md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/TableColumnLineageAPI.java

```diff
@@ -64,18 +88,13 @@ RestWrapper getByTableCol(@RequestParam(value = "tableName", defaultValue = "nul
                 String dot = new String();
                 GetDotForTable getDotForTable = new GetDotForTable();
                 dot = getDotForTable.dotGeneratorWithTable(args);
-                LineageTabColInfo lineageInfo = new LineageTabColInfo();
                 LOGGER.debug(dot);
-                lineageInfo.setDot(dot.toString());
-                lineageInfo.setTableName(tableName);
-                restWrapper = new RestWrapper(lineageInfo, RestWrapper.OK);
+                lineageTabColInfo.setDot(dot.toString());
+//                lineageTabColInfo.setTableName(tableName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)